sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -26,21 +23,67 @@ dotnet add package Azure.AI.OpenAI --prerelease
+You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [OpenAI access](https://learn.microsoft.com/azure/cognitive-services/openai/overview#how-do-i-get-access-to-azure-openai).
 
 ### Authenticate the client
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+In order to interact with the App Configuration service, you'll need to create an instance of the [OpenAIClient][openai_client_class] class. To make this possible, you'll need the endpoint string for your OpenAI resource, as well as your Azure Subscription Key.
+
+#### Get credentials
+
+You can obtain the endpoint string and subscription key from the Azure OpenAI Portal.
+
+#### Create OpenAIClient
+
+Once you have the value of the endpoint string and subscription key, you can create the OpenAIClient:
+
+```C# Snippet:CreateOpenAIClient
+string endpointString = "<endpoint_string>";
+string subscriptionKey = "<azure_subscription_key>";
+var client = new OpenAIClient(new Uri(endpointString), new AzureKeyCredential(subscriptionKey));
+```
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+#### Create OpenAIClient with Azure Active Directory Credential
+
+Client subscription key authentication is used in most of the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][azure_identity_dac] provider shown below,
+or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
+
+```dotnetcli
+dotnet add package Azure.Identity
+```
+
+You will also need to [register a new AAD application][aad_register_app] and [grant access][aad_grant_access] to Configuration Store by assigning the `"App Configuration Data Reader"` or `"App Configuration Data Owner"` role to your service principal.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+```C# Snippet:CreateOpenAIClientTokenCredential
+string endpoint = "<endpoint>";
+var client = new OpenAIClient(new Uri(endpoint), new DefaultAzureCredential());
+```
 
 ## Key concepts
```

Comment:
Maybe we should include a brief explanation of `embeddings` here too. I can prepare it.